js_modules/dagster-ui/packages/ui-core/src/ui/BaseFilters/useFilter.tsx

```diff
@@ -26,28 +27,49 @@ export const FilterTag = ({
   iconName,
   label,
   onRemove,
+  style = 'default',
 }: {
   label: JSX.Element;
   iconName?: IconName;
-  onRemove: () => void;
-}) => (
-  <div>
-    <BaseTag
-      icon={iconName ? <Icon name={iconName} color={Colors.linkDefault()} /> : undefined}
-      rightIcon={
-        <div onClick={onRemove} style={{cursor: 'pointer'}} tabIndex={0}>
-          <Icon name="close" color={Colors.linkDefault()} />
-        </div>
-      }
-      label={label}
-      fillColor={Colors.backgroundBlue()}
-      textColor={Colors.linkDefault()}
-    />
-  </div>
-);
+  onRemove?: () => void;
+  style?: 'default' | 'light-blue';
+}) => {
+  const {color, fillColor, textColor} = useMemo(() => {
+    if (style === 'default') {
+      return {
+        color: Colors.linkDefault(),
+        fillColor: Colors.backgroundBlue(),
+        textColor: Colors.linkDefault(),
+      };
+    } else {
+      return {
+        color: Colors.accentCyan(),
+        fillColor: Colors.backgroundCyan(),
+        textColor: Colors.textCyan(),
+      };
+    }
+  }, [style]);
+  return (
+    <div>
+      <BaseTag
+        icon={iconName ? <Icon name={iconName} color={color ?? Colors.linkDefault()} /> : undefined}
```

Comment:
Looks like `color` should always be defined, so I don't think you need the fallback.